<?xml version="1.0"?>
<launch>
    <group ns="align_to_speaker">
        <rosparam command="load" file="$(find behaviors)/config/2024_align_speaker_config.yaml"/>
        <node name="align_to_speaker_2024" pkg="behaviors" type="2024_shoot_from_subwoofer_server.py.py" output="screen"/>
    </group>
</launch>